src/main/java/nexters/moss/server/application/CakeApplicationService.java

```diff
@@ -1,7 +1,56 @@
 package nexters.moss.server.application;
 
+import nexters.moss.server.application.dto.cake.ArrivalCake;
+import nexters.moss.server.application.dto.cake.CreateANewCakeRequest;
+import nexters.moss.server.application.dto.cake.CreateANewCakeResponse;
+import nexters.moss.server.application.dto.cake.GetANewCakeResponse;
+import nexters.moss.server.application.dto.Response;
+import nexters.moss.server.domain.model.Habit;
+import nexters.moss.server.domain.model.ReceivedPieceOfCake;
+import nexters.moss.server.domain.model.SentPieceOfCake;
+import nexters.moss.server.domain.model.User;
+import nexters.moss.server.domain.repository.HabitRepository;
+import nexters.moss.server.domain.repository.PieceOfCakeReceiveRepository;
+import nexters.moss.server.domain.repository.PieceOfCakeSendRepository;
+import nexters.moss.server.domain.repository.UserRepository;
 import org.springframework.stereotype.Service;
 
+import javax.transaction.Transactional;
+
 @Service
 public class CakeApplicationService {
+    private HabitRepository habitRepository;
+    private UserRepository userRepository;
+    private PieceOfCakeSendRepository pieceOfCakeSendRepository;
+    private PieceOfCakeReceiveRepository pieceOfCakeReceiveRepository;
+
+    public CakeApplicationService(HabitRepository habitRepository, UserRepository userRepository, PieceOfCakeSendRepository pieceOfCakeSendRepository, PieceOfCakeReceiveRepository pieceOfCakeReceiveRepository) {
```

Comment:
이렇게 파라미터 인자가 길 때는
```suggestion
    public CakeApplicationService(
        HabitRepository habitRepository, 
        UserRepository userRepository, 
        PieceOfCakeSendRepository pieceOfCakeSendRepository, 
        PieceOfCakeReceiveRepository pieceOfCakeReceiveRepository
    ) {
```
가 더 가독성 좋은것같아요!